inst/extdata/config_schema.json

```diff
@@ -0,0 +1,191 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "$ref": "#/definitions/Epinow2",
+    "definitions": {
+        "Epinow2": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "job_id": {
+                    "type": "string",
+                    "format": "uuid"
+                },
+                "task_id": {
+                    "type": "string",
+                    "format": "uuid"
+                },
+                "as_of_date": {
```

Comment:
No this would be running the model "as of" a particular date in the past or it could be today. So, we could for example pretend to be running a current version of the model but set "as of" to `2023-11-01` and hypothetically we could select the parameters and data we were using on that date. I was thinking it would be nice for model development -- different model architecture but automatically pull the same data and parameters.

Reply:
So it would look up the parameters we used on that previous as of date? Or we still input the parameters ourselves?

Reply:
Yeah exactly it would just use those parameters. So we could update the parameters for production but still point to the old parameters if we want to compare a new model specification to an old fit.

Reply:
How does it find the parameters from that as of date? It searches through a storage container?

Reply:
Depends on the schema we end up with! But yeah I was thinking it would query the dataset.

Reply:
Have the same question as @natemcintosh. I like the functionality, but seems like you might need somewhere where you specify if the `as_of_date` is using just vintaged data as of that date or also using some type of time stamped parameter sets. Since I think there is a case where you would want to rerun with new parameters, and one case where you'd want to reproduce with old parameters.